working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        mn = 1000000
</pre><p>Why not initialising both to zero? this makes sense anyway (considering our data)..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>